### PR TITLE
Education-Translations (Turkish) (Creating Your User Profile Tutorial)

### PR DESCRIPTION
**Pull request by prepheadrus**

### CODE REVIEW

**Comment by gitforkgit** [comment]
Hey, @prepheadrus  you are having one conflicting file. Maybe this originated from another Pull request that just got merged. Do you know how to resolve the issue?

**Comment by prepheadrus** [comment]
> Hey, @prepheadrus you are having one conflicting file. Maybe this originated from another Pull request that just got merged. Do you know how to resolve the issue?

I fixed ....

File: Projects/Community-Plugins/Schemas/Docs-Nodes/P/Plugins/Plugins/plugins.json [hunk 1]
```diff
@@ -9,14 +9,9 @@
...
-                "text": "Η ιεραρχία των πρόσθετων καθορίζει ποιά πρόσθετα θα συμπεριληφθούν στον τρέχοντα χώρο εργασίας. Στο πλαίσιο αυτής της ιεραρχίας, τα πρόσθετα είναι αρχεία JSON που διαθέτουν την περιγραφή μιας ιεραρχίας όπως ένα εξορυκτήριο δεδομένων, ένα εξορυκτήριο συναλλαγών, μια μηχανή συναλλαγών ή ένα σύστημα συναλλαγών.",
-                "updated": 1641828795260
-            },
-            {
-                "language": "DE",
-                "text": "Die Plugin-Hierarchie definiert, welche Plugins in den aktuellen Arbeitsbereich aufgenommen werden sollen. Im Kontext dieser Hierarchie sind Plugins JSON-Dateien mit der Beschreibung einer Hierarchie, z. B. einer Data Mine, Trading Mine, Trading Engine oder eines Trading Systems.",
-                "updated": 1641764838784
+                "language": "TR",
+                "text": "Eklentiler hiyerarşisi, geçerli çalışma alanına hangi eklentilerin dahil edileceğini tanımlar. Bu hiyerarşi bağlamında eklentiler, veri madeni, ticaret madeni, ticaret motoru veya ticaret sistemi gibi bir hiyerarşiyi tanımlayan JSON dosyalarıdır.",
+                "updated": 1644582475185
```
Comment (gitforkgit) [inline Projects/Community-Plugins/Schemas/Docs-Nodes/P/Plugins/Plugins/plugins.json:14]: It still shows me that thre spanish and german translation gets overwritten.

**Review by gitforkgit — CHANGES_REQUESTED** [review]
This file still shows up with the spanish and german translation getting overwritten.

File: Projects/Community-Plugins/Schemas/Docs-Nodes/P/Plugins/Plugins/plugins.json (hunk 1 above)
Comment (prepheadrus) [reply Projects/Community-Plugins/Schemas/Docs-Nodes/P/Plugins/Plugins/plugins.json:14]: > It still shows me that thre spanish and german translation gets overwritten.

I don't know how to fix this. what should I do?

Comment (maxah) [reply Projects/Community-Plugins/Schemas/Docs-Nodes/P/Plugins/Plugins/plugins.json:14]: image
It should be like this

Comment (prepheadrus) [reply Projects/Community-Plugins/Schemas/Docs-Nodes/P/Plugins/Plugins/plugins.json]: thanks maxah

Comment (prepheadrus) [reply Projects/Community-Plugins/Schemas/Docs-Nodes/P/Plugins/Plugins/plugins.json]: > It still shows me that thre spanish and german translation gets overwritten.
can you check? is there any error?

Comment (maxah) [reply Projects/Community-Plugins/Schemas/Docs-Nodes/P/Plugins/Plugins/plugins.json]: Save the file in Superalgos\Projects\Community-Plugins\Schemas\Docs-Nodes\P\Plugins\Plugins\plugins.json and run app.contribute
[plugins.txt](https://github.com/Superalgos/Superalgos/files/8119732/plugins.txt)

Comment (prepheadrus) [reply Projects/Community-Plugins/Schemas/Docs-Nodes/P/Plugins/Plugins/plugins.json]: > It still shows me that thre spanish and german translation gets overwritten.

seems to be fixed now

**Comment by gitforkgit** [comment]
@prepheadrus The file now seems to be fixed but there is one conflict in: Projects/Community-Plugins/Schemas/Docs-Nodes/P/Plugin/Plugin-Project/plugin-project.json
That may come from a merge of another pull request. If you have that fixed, please leave me a message :+1:

**Comment by prepheadrus** [comment]
> @prepheadrus The file now seems to be fixed but there is one conflict in: Projects/Community-Plugins/Schemas/Docs-Nodes/P/Plugin/Plugin-Project/plugin-project.json That may come from a merge of another pull request. If you have that fixed, please leave me a message 👍

I fixed it, I hope there are no other problems.

**Comment by gitforkgit** [comment]
@prepheadrus I will look trough again :+1:

File: Projects/Community-Plugins/Schemas/Docs-Nodes/P/Plugin/Plugin-Project/plugin-project.json [hunk 2]
```diff
@@ -33,6 +41,10 @@
             "updated": 1609869554268,
             "translations": [
                 {
+                    "language": "TR",
+                    "text": "Özellikler",
+                    "updated": 1645082938811
+                  {
                     "language": "RU",
                     "text": "Свойства",
```
Comment (gitforkgit) [inline Projects/Community-Plugins/Schemas/Docs-Nodes/P/Plugin/Plugin-Project/plugin-project.json:49]: There is the closing bracket with a comma after it missing.
},
{

You can see how on other files.

**Review by gitforkgit — CHANGES_REQUESTED** [review]
I dont know if other files have this issue too, so please look trough. If this issues are fixed it could get merged as far as I can tell. :+1:

File: Projects/Community-Plugins/Schemas/Docs-Nodes/P/Plugin/Plugin-Project/plugin-project.json (hunk 2 above)
Comment (prepheadrus) [reply Projects/Community-Plugins/Schemas/Docs-Nodes/P/Plugin/Plugin-Project/plugin-project.json:47]: I hope it's ok..I will not translate again.. Promise :=)))

Comment (gitforkgit) [reply Projects/Community-Plugins/Schemas/Docs-Nodes/P/Plugin/Plugin-Project/plugin-project.json:47]: @prepheadrus I hope will keep translating. You have translated a lot and nearly all files look great and are adding value to superalgos. I think i didn`t exactly tell you where to add:
},
{
Thats my fault. Just so i can merge this pull request and by that you getting rid of troubles, can you replace the conflicting file by the one from the repository and app.contribute then? By this the "original" version gets pushed and the conflicting file gets replaced. Later on you can add the translation of this one file back.
We are very close to target :+1:

Comment (prepheadrus) [reply Projects/Community-Plugins/Schemas/Docs-Nodes/P/Plugin/Plugin-Project/plugin-project.json]: I downloaded the original file from the develop branch and am sending it again. I hope it works this time.

File: Projects/Community-Plugins/Schemas/Docs-Nodes/P/Plugin/Plugin-Project/plugin-project.json [hunk 3]
```diff
@@ -1,55 +1,2976 @@
...
+
+  
+  
+  <script crossorigin="anonymous" defer="defer" integrity="sha512-vkftKchtwD8WsIMRv6keDt+WfX8+SAcIsUPCrzBbL2ojw8iJ1qck2x+XFSii9xkMywZvwKOvVkGMdm09IUjmlg==" type="application/javascript" src="https://github.githubassets.com/assets/environment-be47ed29.js"></script>
+<script crossorigin="anonymous" defer="defer" integrity="sha512-PkG/XW9tJlpl1ddcDh07L65WNnaJOuScqwUeBHgAm3dGk5rhHGUPOp4za+WKVBzzwDkkV1m1EJDdrKhsNKoUtA==" type="application/javascript" src="https://github.githubassets.com/assets/chunk-frameworks-3e41bf5d.js"></script>
+<script crossorigin="anonymous" defer="defer" integrity="sha512-3ufRKZJe/1/AvITVwGVuNFv3n9rbKlZcpZkfa2IwVFwI37H7AC45jYGBDi0LrlaY0Kdyb6YzEjclzP+lSjJx9Q==" type="application/javascript" src="https://github.githubassets.com/assets/chunk-vendor-dee7d129.js"></script>
+<script crossorigin="anonymous" defer="defer" integrity="sha512-vgWV8b1FcpGErHwOitpIbvNmtQZwsw55TUraZngdfnHB3P+ZQnxzKD1L//RqzlWMB9sExBRhfOt1rHCxrru/Fg==" type="application/javascript" src="https://github.githubassets.com/assets/github-elements-be0595f1.js"></script>
```
[the remark this answers is not in the document]
Comment (prepheadrus) [reply Projects/Community-Plugins/Schemas/Docs-Nodes/P/Plugin/Plugin-Project/plugin-project.json]: The only thing I did at the beginning was to translate a document as usual.. I couldn't handle it for 9 days. I didn't do anything other than what I've done for months this time.

[the remark this answers is not in the document]
Comment (prepheadrus) [reply Projects/Community-Plugins/Schemas/Docs-Nodes/P/Plugin/Plugin-Project/plugin-project.json]: Please don't misunderstand what I wrote. This has nothing to do with you, man. I'm just trying to understand why what I always do is so complicated this time. I'm sorry if I upset you.

**Review by gitforkgit — APPROVED** [review]

File: Projects/Community-Plugins/Schemas/Docs-Nodes/P/Plugin/Plugin-Project/plugin-project.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (prepheadrus) [reply Projects/Community-Plugins/Schemas/Docs-Nodes/P/Plugin/Plugin-Project/plugin-project.json]: hey, we are people working for the same thing. we are members of the same community..we all know what we do is for superalgos.i have to say it again, what i wrote has nothing to do with you man..we are good together.

**Pull request merged**